pkg/admission/namespacelifecycle/admission.go

```diff
@@ -75,15 +75,15 @@ func newWorkspaceNamespaceLifecycle() (*workspaceNamespaceLifecycle, error) {
 		return nil, err
 	}
 
-	lifecycle, err := lifecycle.NewLifecycle(sets.NewString())
```

Comment:
rever. Not worth the effort.

Reply:
done, but what is about alias `lc` for the imported package?